bundles/org.openhab.binding.omatic/LICENSE

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
The legal things are already covered by the main project and the header in each of your files. So, this could be removed.

Reply:
Fixed in https://github.com/openhab/openhab-addons/pull/9286/commits/ebd0ae674be0e8107722de21ba936c75db3d6f20